packages/web/src/components.d.ts

```diff
@@ -390,26 +390,34 @@ export namespace Components {
         "hintId": string;
     }
     interface GcdsIcon {
+        /**
+          * Style of the icon.
+         */
+        "iconStyle"?: 'regular' | 'solid';
```

Comment:
Regular is outlined icons
Solid is filled

Reply:
The solid style has most icons. Regular is mostly outlined variations for solid icons.

Reply:
What about using `outlined` instead of `regular`? Makes it a little bit easier to understand the difference with `solid` icons.

Reply:
Good point, in that case, could you please add a simple description to the prop comment?
Something very easy like "the regular style, will use the outlined version of the icon" or something like that.